public_suffix_list.dat

```diff
@@ -14312,6 +14312,11 @@ net.ru
 org.ru
 pp.ru
 
+// Mysten Labs : https://mystenlabs.com
+// Submitted by Reginaldo Silva <reginaldo.silva@mystenlabs.com>
```

Comment:
Please use a non-personal email address so we can contact your organization even if you leave.